src/main/java/com/fasterxml/jackson/databind/jdk14/JDK14Util.java

```diff
@@ -28,9 +27,8 @@ public static String[] getRecordFieldNames(Class<?> recordType) {
         return RecordAccessor.instance().getRecordFieldNames(recordType);
     }
 
-    public static AnnotatedConstructor findRecordConstructor(DeserializationContext ctxt,
-            BeanDescription beanDesc, List<String> names) {
-        return new CreatorLocator(ctxt, beanDesc)
+    public static AnnotatedConstructor findRecordConstructor(AnnotatedClass recordClass, AnnotationIntrospector intr, MapperConfig<?> config, List<String> names) {
```

Comment:
API change? Possibly not a good thing - any chance that the old version can be kept and deprecated?

Reply:
Yeah no, please leave the signature as it was: `DeserializationContext` and `BeanDescription` can be used to access `AnnotationIntrospector` and `MapperConfig`.

Reply:
The problem is in the place I'm using it now (`POJOPropertiesCollector`), I don't have `DeserializationContext` nor `BeanDescription`...

Reply:
Hmmm, ok that makes sense. But that's bit of a problem. Let me think about it.